src/icrar/leap-accelerate/algorithm/casa/PhaseRotate.cc

```diff
@@ -188,8 +188,9 @@ namespace casalib
                 dInt = 0;
                 for(size_t n = 0; n < metadata.I.size(); ++n)
                 {
-                    dInt.row(n) = avg_data_slice.row(n) - casacore::sum(metadata.A.row(n) * cal1.column(0)); 
+                    dInt.row(n) = avg_data_slice.row(n) - (casacore::sum((casacore::Array<double>)metadata.A.row(n) * (casacore::Array<double>)cal1.column(0)));
```

Comment:
These casts seem unnecessary (and weren't there before). You also don't need to quality `sum` with `casacore`, because its argument already lives in that namespace. Plus there was an extra set of parenthesis.

I think all in all this could read:
`dInt.row(n) = avg_data_slice.row(n) - sum(metadata.A.row(n) * cal1.column(0));`

Reply:
I updated here to remove this troublesome warning:
```
PhaseRotate.cc:191:96: warning: ISO C++ says that these are ambiguous, even though the worst conversion for the first is better than the worst conversion for the second:
  191 |                     dInt.row(n) = avg_data_slice.row(n) - sum(metadata.A.row(n) * cal1.column(0));
```

Reply:
My best guess at the reason behind it is that there are two add implementations that the compiler can't decide between, e.g.:
* casacore::Array<T> + casacore::Array<T>
* casacore::Vector<T> + casacore::Vector<T>